### PR TITLE
libprotobuf v5.26.0

### PR DESCRIPTION
**Pull request by h-vetinari**
Currently on rc3; bunch of breaking changes causing major version bump see https://protobuf.dev/news & https://github.com/protocolbuffers/protobuf/commit/16afb4dea76efab652d86e4ebf9a118138f88578. The Python breaking changes look painful, but that's a question for the other feedstock...

~Based on #208, will be rebased once that's in.~

### CODE REVIEW

**Comment by xhochy** [comment]
I get some failures on `osx-arm64` but there, I see more output. Sadly, this doesn't happen on win.

**Comment by h-vetinari** [comment]
Now fixed and ready for review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,16 @@
 # keep this without major version to let the bot pick it up
-{% set version = "25.3" %}
+{% set version = "26.0" %}
 # protobuf doesn't add the major version in the tag, it's defined per language in
 # https://github.com/protocolbuffers/protobuf/blob/main/version.json
-{% set major = "4" %}
+{% set major = "5" %}
+# pkg-config metadata sets yet another version format, not using the major version
+# but adding the rc number as a patch version, or 0 for regular releases
```
Comment (jjerphan) [inline recipe/meta.yaml:7]: Is there some documentation about it somewhere?

**Review by jjerphan — APPROVED** [review]
Thank you, @xhochy and @h-vetinari.

I just have one comment.

File: recipe/meta.yaml (hunk 1 above)
Comment (jjerphan) [reply recipe/meta.yaml:7]: I find this behavior dangerous. :eyes:

Comment (h-vetinari) [reply recipe/meta.yaml:7]: I'm pretty sure it's the way protobuf has set up their metadata; in any case, we're doing exact matches on the versions (according to the pkg-config metadata) in the testing here, so I'm not too worried.

**Review by xhochy — APPROVED** [review]

**Pull request merged**